### PR TITLE
Add integrations endpoints

### PR DESCRIPTION
**Pull request by q-b**
This PR adds integrations (webhooks) endpoints.
I took a shortcut and copy/pasted `call_api_post()` to create `call_api_put()` since it is needed for the `integrations.update` endpoint. It could probably be improved later, but the current version should be sufficient for now.

### CODE REVIEW

File: tests/test_integrations.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    integration_webhook_incoming = integrations_create_webhook_incoming
+    assert integration_webhook_incoming.get("success")
+
+    integration_webhook_outgoing = logged_rocket.integrations_create(
+        integrations_type="webhook-incoming",
+        name=str(uuid1()),
+        enabled=True,
+        event="sendMessage",
+        username=logged_rocket.me().json().get("username"),
+        urls=["http://example.com/fake"],
```
Comment (jadolg) [inline tests/test_integrations.py:28]:
```suggestion
        urls=["https://example.com/fake"],
```

**Comment by jadolg** [comment]
Looks like github actions are "acting weird" today. I'll give them another try later.

File: rocketchat_API/APISections/base.py [hunk 2]
```diff
@@ -114,6 +114,40 @@ def call_api_post(self, method, files=None, use_json=None, **kwargs):
             timeout=self.timeout,
         )
 
+    def call_api_put(self, method, files=None, use_json=None, **kwargs):
+        reduced_args = self.__reduce_kwargs(kwargs)
+        # Since pass is a reserved word in Python it has to be injected on the request dict
+        # Some methods use pass (users.register) and others password (users.create)
+        if "password" in reduced_args and method != "users.create":
+            reduced_args["pass"] = reduced_args["password"]
+            del reduced_args["password"]
```
Comment (jadolg) [inline rocketchat_API/APISections/base.py:123]: Maybe let's get rid of this. It's not used and not tested. Let's introduce it when needed if needed.

File: rocketchat_API/APISections/integrations.py [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+        enabled,
+        username,
+        channel,
+        script_enabled,
+        event=None,
+        urls=None,
+        **kwargs
+    ):
+        """Creates an integration."""
+        if integrations_type == "webhook-outgoing":
```
Comment (jadolg) [inline rocketchat_API/APISections/integrations.py:19]: Is there any way we could test this path?

File: tests/test_integrations.py (hunk 1 above)
Comment (jadolg) [inline tests/test_integrations.py:28]:
```suggestion
        urls=["https://example.com/fake"],
```
Just to make SonarCloud happy

**Review by jadolg — CHANGES_REQUESTED** [review]
Just a couple of details were spotted by the automated tools.

File: rocketchat_API/APISections/integrations.py (hunk 3 above)
Comment (q-b) [reply rocketchat_API/APISections/integrations.py:19]: Creation is tested below the incoming one : https://github.com/jadolg/rocketchat_API/blob/3c4cfb970baabfec5d9091f8335d0392fafe275e/tests/test_integrations.py#L22-L32

I did not bother with the others functions, I don't think they are relevant here (maybe the update one?) and add much value.

Comment (q-b) [reply rocketchat_API/APISections/integrations.py:19]: Ok while posting I saw my mistake, I forgot to change the `integrations_type` :man_facepalming: 
I'll fix it.

**Review by jadolg — APPROVED** [review]
LGTM

releasing now :)

**Pull request merged**